docs/companion_guide.md

```diff
@@ -241,7 +241,172 @@ on certain terms and concepts.
+| ---------------------------- | -------------------------------------------- |
+| Description                  | Ceramic is a decentralized data network. Its foundations are laid on top of the Ceramic Event Driven Protocol. The infrastructure to build Personal Data Stores is offered by the Ceramic ComposeDB. ComposeDB replaces IDX and DID Data Store. |
+| Specification                |	Open Specification curated by Ceramic.Network |
+| Deployment                   |	The ComposeDB instance is installed as part of the Ceramic Node deployment. It can only be hosted in a Cloud environment. |
+| Identity                     | Decentralized Identifiers DID's. |
+| Authentication               |	Web3 Wallets and DID. |
+| Authorization	               | Object Capabilities |
+| Transport                    | GraphQL API over HTTP/1.1 |
+| Schema / Data Representation |	API models are defined as GraphQL Schemas. The underlying data store uses graph nodes: Accounts and Documents. Relations are expressed as Edges. |
+| Query Capabilities           | Partial GraphQL Queries. As of this writing, a query cannot be made against any data attributes. |
+
+| Atomic Data and Atomic Server | https://docs.atomicdata.dev/ |
+| ----------------------------- | ------------------------------------------- |
+| Description	                  | Atomic offers a specification and a server to build Json-LD for building privacy preserving applications. |
+Specification	                  | Open-Source Specification. The Atomic Server implementation in Rust is also open sourced. |
+| Deployment                    | It can be deployed in a Cloud environment or User-Hosted |
+| Identity                      | PKI |
+| Authentication                |	Json-AD Authentication Resource |
+| Authorization                 |	Atomic Hierarcy Model |
+| Transport	                    | WebSockets, HTTP 1/1 |
+| Schema / Data Representation  |	Json-AD (Jason-Atomic Data). A variation to Json-LD but Atomic Data supports the definition of schemas to provide type-safey. |
+| Query Capabilities            |	Atomic Paths, SPARQL. |
+
+
+| Encrypted Data Vaults | https://identity.foundation/edv-spec/ |
+| --------------------- | --------------------------------------------------- |
+| Description           |	A specification with the goal of ensuring the privacy of an entity’s data by encrypting the data at rest. |
+| Specification         |	Open-Source Specification incubated by DIF. |
+| Deployment            |	[Pending] |
+| Identity              | Support for various Identity models. DIDs being one of them. |
+| Authentication        |	[Pending] |
+| Authorization         |	Authorization Capabilities |
+| Transport             |	HTTP 1/1, gRPC, Bluetooth |
+| Schema / Data Representation | [Pending] |
+| Query Capabilities | The goal is to provide Indexing and Querying capabilities. The working group is in the process of how deciding how this is done. |
```

Comment:
```suggestion
| Query Capabilities | The goal is to provide Indexing and Querying capabilities. The working group is in the process of how deciding how this will be done. |
```